packages/terra-notification-dialog/src/terra-dev-site/doc/example/ReversedActionNotificationDialog.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from 'terra-button';
+import NotificationDialog, { NotificationDialogVariants } from '../../../NotificationDialog';
+
+const clickConfirm = () => {
+  console.log('You clicked confirm'); // eslint-disable-line no-console
+};
+
+class ReversedActionNotificationDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
```

Comment:
Could you change these new examples to use hooks?

Reply:
Have done so, also went ahead and updated the older examples
https://github.com/cerner/terra-framework/pull/899/commits/35d2d0ec00602f21d5d27309d72d297d8ed9d925